tools/WrapExecutableLinker.sh

```diff
@@ -7,13 +7,13 @@ temp_files=()
 trap 'rm -r "${temp_files[@]}"' EXIT
 
 pushd @CMAKE_SOURCE_DIR@ >/dev/null
-git_description="@GIT_DESCRIPTION@"
-git_branch="@GIT_BRANCH@"
-if [ -z "$git_description" ]; then
-    git_description="NOT_IN_GIT_REPO"
```

Comment:
Could you verify, if you haven't, that this doesn't break formaline? I think we have a test that the repo from the HDF5 files can be built, but this looks like a change that really could break that.

Reply:
Seems to work in the container, and the formaline test on CI also passes